definitions/infra-kubernetespod/definition.yml

```diff
@@ -0,0 +1,35 @@
+domain: INFRA
+type: KUBERNETES_POD
+configuration:
+  entityExpirationTime: DAILY
+  alertable: true
+
+synthesis:
+  rules:
+  - identifier: entity.id
+    name: k8s.podName
+    encodeIdentifierInGUID: true
+    conditions:
+    - attribute: k8s.pod.isReady
+    tags:
+      k8s.status:
+      k8s.clusterName:
+      k8s.namespaceName:
+      k8s.podName:
+      k8s.nodeName:
+      k8s.pod.createdAt:
+      k8s.pod.startTime:
+      k8s.pod.isReady:
+      k8s.pod.isScheduled:
```

Comment:
Shouldn't these two be `multiValue: false` ? Same probably with status

Reply:
I don't know. In the definition of the INFRA Container entity there's no multivalue. Maybe it's missing something there?

Reply:
[Here's](https://github.com/newrelic-experimental/entity-synthesis-definitions#tags) an example btw

Reply:
@douglascamata yep we are probably missing them there too. Since the multiValue support was added not long ago here.